ui/src/app/pages/analysis/interactive-notebook.spec.tsx

```diff
@@ -112,13 +114,25 @@ test.each([
     renderInteractiveNotebook(pathParameters);
     const editButton = screen.getByTitle('Edit');
     await user.click(editButton);
-    await waitFor(() => {
-      expect(spyWindowOpen).toHaveBeenCalledTimes(1);
-      expect(spyWindowOpen).toHaveBeenCalledWith(
-        app.proxyUrls[GKE_APP_PROXY_PATH_SUFFIX],
-        '_blank'
-      );
-    });
+    if (appType !== 'RStudio') {
```

Comment:
"if not ... else" is a little awkward - how about listing the "= RStudio" case first?

Reply:
done